styles.css

```diff
@@ -31,9 +32,173 @@ body {
   border-radius: 18px;
   border: 6px solid var(--clr-border2);
   width: 550px;
-  margin-top: 50px;
+  margin: 50px auto;
 }
 
 .description {
-  text-align: center;        
+  text-align: center; 
+  margin-left: 10px;  
+  margin-right: 10px;     
+}
+
+.title {
+  text-align: center;
+  margin-top: 50px;
+  margin-bottom: 25px;
+}
+
+.progress {
+  height: 500px;
+  width: 50%;
+  overflow: hidden;
+  display: flex;
+  justify-content: space-around;
+  align-content: center;
+  flex-flow: row wrap;
+  margin: 0 auto;
+  margin-top: 15px;
+  margin-bottom: 50px;
+  gap: 20px;
+}
+
+.skill {
+  width: 160px;
+  height: 160px;
+  position: relative;
+}
+
+.outer {
+  height: 160px;
+  width: 160px;
+  margin-top: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 6px 6px 10px -1px rgba(0, 0, 0, 0.15)
+  , -6px -6px 10px -1px rgba(255, 255, 255, 0.7);
+  border-radius: 50%;
+}
+
+.inner {
+  height: 80px;
+  width: 80px;
+  padding: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column-reverse;
+  border-radius: 50%;  
+  box-shadow: inset 4px 4px 6px -1px rgba(0, 0, 0, 0.2)
+  , inset -4px -4px 6px -1px rgba(255, 255, 255, 0.7),
+  0.5px 0.5px 0px rgba(0, 0, 0, 0.15),
+   0px 12px 10px -10px rgba(0, 0, 0, 0.05);
+}
+
+.percentage {
+  font-weight: 60;
+  color: #555;
+  font-size: 20px;
+}
+
+.lenguage {
+  font-weight: 600;
+  color: #555;
+  font-size: 25px;
+}
+
+.circleangular svg circle {
+  fill: none;
+  stroke: red;
+  stroke-width: 20px;
+  stroke-dasharray: 450;
+  stroke-dashoffset: 450;
+  animation: animang 0.5s linear forwards;
+}
+
+.circlehtml svg circle {
+  fill: none;
+  stroke: darkred;
+  stroke-width: 20px;
+  stroke-dasharray: 450;
+  stroke-dashoffset: 450;
+  animation: animhtml 2s linear forwards;
+}
+
+.circlecss svg circle {
+  fill: none;
+  stroke: blue;
+  stroke-width: 20px;
+  stroke-dasharray: 450;
+  stroke-dashoffset: 450;
+  animation: animcss 2s linear forwards;
+}
+
+.circlejs svg circle {
+  fill: none;
+  stroke: yellow;
+  stroke-width: 20px;
+  stroke-dasharray: 450;
+  stroke-dashoffset: 450;
+  animation: animjs 2s linear forwards;
+}
+
+.circlec svg circle {
+  fill: none;
+  stroke: darkblue;
+  stroke-width: 20px;
+  stroke-dasharray: 450;
+  stroke-dashoffset: 450;
+  animation: animc 2s linear forwards;
+} 
+
+.circlejava svg circle {
+  fill: none;
+  stroke: #12A2E7;
+  stroke-width: 20px;
+  stroke-dasharray: 450;
+  stroke-dashoffset: 450;
+  animation: animjava 2s linear forwards;
+}
```

Comment:
You could create a common class with the common styles, like:

.commonClass {
  fill: none;
  stroke-width: 20px;
  stroke-dasharray: 450;
  stroke-dashoffset: 450;
}

Then, in every specific class keep the things that changes:

.circleangular svg circle {
  stroke: red;
  animation: animang 0.5s linear forwards;
}